modules/product/src/Tests/ProductTest.php

```diff
@@ -34,7 +34,8 @@ function testAddCommerceProductExistingSku() {
     $values = [
       'sku' => $this->randomMachineName(),
       'title' => $this->randomMachineName(),
-      'type' => 'product'
+      'type' => 'product',
+      'store_id' => $this->commerce_store->id()
```

Comment:
Yes, but let's do that on https://github.com/commerceguys/commerce/pull/192 to prevent making https://github.com/commerceguys/commerce/pull/192 a pain to merge.